docker-compose.yml

```diff
@@ -1,91 +1,85 @@
 version: "3.9"
 services:
   mysql:
-    image: mysql:8.0.27
-    command: --default-authentication-plugin=mysql_native_password
+    image: mysql:5.7
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: mysql_root_password
-      MYSQL_USER: root
       MYSQL_DATABASE: app
-    ports:
-      - "33061:3306"
+    expose:
+      - "3306"
     volumes:
       - mysql:/var/lib/mysql
-    logging:
-      driver: none
   rabbitmq:
     image: rabbitmq:3.9.12
     restart: always
     environment:
       RABBITMQ_NODENAME: rabbit
-    ports:
-      - "5672:5672"
-      - "15672:15672"
-      - "25672:25672"
-      - "4369:4369"
+    expose:
+      - "5672"
+      - "15672"
+      - "25672"
+      - "4369"
   memcached:
     image: memcached:1.6.12
     restart: always
-    ports:
-      - "11211:11211"
+    expose:
+      - "11211"
   mongodb:
     image: mongo:5.0.4
     restart: always
     volumes:
       - /data/db
-    ports:
-      - "27017:27017"
+    expose:
+      - "27017"
   postgres:
-    image: postgres:14.1
+    image: postgres:14.2
     restart: always
     environment:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: postgres_password
     volumes:
-      - ./backups:/home/backups
-    ports:
-      - "5432:5432"
+      - postgres:/var/lib/postgresql/data
+    expose:
+      - "5432"
   redis:
     image: redis:6.2.6
     restart: always
     volumes:
       - /var/lib/redis
-    ports:
-      - "6379:6379"
+    expose:
+      - "6379"
   app:
-    build: .
-    image: newrelic/newrelic-ruby-agent-tester
+    build:
+      context: .
+      args:
+        - ruby_version=${RUBY_VERSION}
+    image: rpm
```

Comment:
I like this simplicity here. I wonder if we should go straight for the gem name in that case? `newrelic_rpm`? I'm already getting it confused with RPM site.

Reply:
Good call. We want to disambiguate from RedHat too! Addressed with [ca0c5e3](https://github.com/newrelic/newrelic-ruby-agent/pull/963/commits/ca0c5e337b2b032cf1dfdbe84a924fe23a89ba35)